tests/integration/reusable_values.py

```diff
@@ -39,6 +47,38 @@ async def _set_up_reusable_values():
         WALLET,
     )
 
+    await sign_and_reliable_submission_async(
```

Comment:
I would do all of this with an account created just for the clawback test rather than editing the globally-used wallets.

Reply:
This can go in the clawback test instead of at the very beginning - there's only one test that needs this, so it doesn't need to be a global variable

Reply:
could i call `fund_wallet` inside the the individual integration test though? 

When I tried to move the logic into my test, i got  `<string>:4: RuntimeWarning: coroutine 'fund_wallet' was never awaited`, even though I did call `await` in front of it.

Reply:
This PR should fix that: https://github.com/XRPLF/xrpl-py/pull/619

The tests are run on every client version (sync and async, JSON RPC and websocket), so sometimes the sync code runs into issues.

Reply:
Just merged, you should be good now :)

Reply:
Thanks!

Reply:
@mvadari  I got `RuntimeWarning: coroutine 'sign_and_submit' was never awaited`

Reply:
@mvadari  I got an error for `sign_and_submit` now :(

```
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/shawnxie/Desktop/xrpl-py/tests/integration/it_utils.py", line 287, in modified_test
    asyncio.run(
  File "/Users/shawnxie/.pyenv/versions/3.9.1/lib/python3.9/asyncio/runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "/Users/shawnxie/.pyenv/versions/3.9.1/lib/python3.9/asyncio/base_events.py", line 642, in run_until_complete
    return future.result()
  File "/Users/shawnxie/Desktop/xrpl-py/tests/integration/it_utils.py", line 272, in _run_async_test
    raise e
  File "/Users/shawnxie/Desktop/xrpl-py/tests/integration/it_utils.py", line 268, in _run_async_test
    await test_function(self, client)
  File "/Users/shawnxie/Desktop/xrpl-py/tests/integration/transactions/test_clawback.py", line 25, in test_basic_functionality
    await fund_wallet(HOLDER)
  File "/Users/shawnxie/Desktop/xrpl-py/tests/integration/it_utils.py", line 103, in fund_wallet
    sign_and_submit(payment, client, MASTER_WALLET, check_fee=True)
  File "/Users/shawnxie/Desktop/xrpl-py/xrpl/transaction/main.py", line 34, in sign_and_submit
    return asyncio.run(
  File "/Users/shawnxie/.pyenv/versions/3.9.1/lib/python3.9/asyncio/runners.py", line 33, in run
    raise RuntimeError(
RuntimeError: asyncio.run() cannot be called from a running event loop

----------------------------------------------------------------------
Ran 1 test in 11.120s

FAILED (errors=2)
sys:1: RuntimeWarning: coroutine 'sign_and_submit' was never awaited
```

Reply:
nevermind i fixed it